mne/decoding/transformer.py

```diff
@@ -111,13 +111,17 @@ def __init__(self, info=None, scalings=None, with_mean=True,
         self.with_std = with_std
         self.scalings = scalings
 
+        if (scalings is None) and (info is None):
+            raise ValueError('You have to specify either "info" or'
+                             '"scaling". Currently both are'
+                             '{}'.format(type(info)))
```

Comment:
The `'{}'.format()` is a bit extravagant in this specific case, isn't it? Can just do ```"your text here" + str(type(info))```

Reply:
Mmmh yes. Are there any caveats associated with either approach? I tend to always use the .format for consistency ...
- '{}'.format(x)
- ('%' % x)
- '' + str(x)

Reply:
"Need to specify info if `scalings` is None"

Reply:
I like `str.format` myself,  but in this case you're adding a lot of syntax that's not really necessary I would say. But it's ok either way.